wallet-new/src/Cardano/Wallet/Kernel/DB/TxMeta/Types.hs

```diff
@@ -265,15 +276,22 @@ data FilterOrdering =
     | LesserThanEqual
     deriving (Show, Eq, Enum, Bounded)
 
+data PutReturn = Tx | Meta | No
+    deriving (Show, Eq, Enum, Bounded)
```

Comment:
I don't understand what this type signifies. This needs a comment, both for the type and for each constructor.